docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md

```diff
@@ -33,13 +33,13 @@ Using the **Edit Universe** page, you can modify the following:
 - Regions - you can select any region configured in the provider used to deploy the universe
 - Number of nodes and Availability zones - as you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones
 - [Master placement](../../create-deployments/dedicated-master/)
-- Instance type and volume size - you can select instance types configured in the provider
+- Instance type, Volume count and size - you can select instance types configured in the provider
+- Replication Factor (RF) - RF increases are supported for both k8s and VM universes. While editing RF, the user can add new AZs but cannot change other universe settings such as instance type
+- Storage Type
 - [User tags](../instance-tags/)
 
 YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
 
-Note that you can't change the replication factor of a universe.
```

Comment:
I guess we should keep this sentence but rewrite it as this affects only primary cluster